Fix and improve template pages documentation (#229)
exoplatform/exo-documentation: docs/Customizing_site.rst

Creating a page layout template
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> In eXo Platform, a page is a set of portlets.
> The portlets can be arranged in columns, rows and tabs.

Reviewer: ```suggestion
In eXo Platform, a page is a set of portlets which could be arranged in columns, rows, and tabs.
```

.. |image4| image:: images/results_unifiedsearch_wikipedia.png
.. |image5| image:: images/search_results_notwikipedia.png
.. |image6| image:: images/page_layout/template.png

> .. |image7| image:: images/page_layout/extension_structure.png

Reviewer: Looks like you forgot to add the image

Author: No, the image is already there, but it was not referenced in the doc : https://github.com/exoplatform/exo-documentation/blob/master/docs/images/page_layout/extension_structure.png
